ruler.go

```diff
@@ -1220,6 +1212,11 @@ A tool by @_staaldraad from @sensepost to abuse Exchange Services.`
 			Name:  "rpc",
 			Usage: "Force RPC/HTTP rather than MAPI/HTTP",
 		},
+		cli.StringFlag{
+			Name:  "hostname,n",
+			Value: "",
```

Comment:
Could we leave the default value as RULER. 

It is one of the IOCs we published for the tool and although easily modifiable, it is worth having as a "script kiddie" or basic detection effort built into the tool. 

https://github.com/sensepost/notruler/blob/master/iocs.md#local-host-domain-controllerexchange-server

Reply:
Sure, I forgot to set it as the default value